### PR TITLE
fix HWMON

### PR DESCRIPTION
**Pull request by saiaman**
There was a typo for HWMON it's fixed now.

### CODE REVIEW

**Review by Djadih — APPROVED** [review]

File: cspell.json [hunk 1]
```diff
@@ -117,5 +117,8 @@
     "solcx",
     "TSTK"
   ],
-  "flagWords": ["hte", "addresss"]
-}
+  "flagWords": [
+    "hte",
+    "addresss"
```
Comment (Djadih) [inline cspell.json:122]: Is this a typo?

**Review by Djadih — CHANGES_REQUESTED** [review]
Retracted for comment

File: cspell.json (hunk 1 above)
Comment (saiaman) [reply cspell.json:122]: No just a nicer format of JSON file :)

Comment (saiaman) [reply cspell.json:122]: Same data just formatted

Comment (saiaman) [reply cspell.json:122]: yes i have enabled auto-format onSave sorry

**Review by Djadih — APPROVED** [review]

**Review by Juuddi — APPROVED** [review]

**Pull request merged**